e2e-tests/cypress/e2e/journeys/insurance/account/create/resend-confirm-email/resend-confirm-email.spec.js

```diff
@@ -0,0 +1,59 @@
+import { confirmEmailPage } from '../../../../../pages/insurance/account/create';
+import { PAGES } from '../../../../../../../content-strings';
+import { INSURANCE_ROUTES as ROUTES } from '../../../../../../../constants/routes/insurance';
+import api from '../../../../../../support/api';
+
+const CONTENT_STRINGS = PAGES.INSURANCE.ACCOUNT.CREATE.CONFIRM_EMAIL_RESENT;
+
+const {
+  START,
+  ACCOUNT: { CREATE: { CONFIRM_EMAIL, CONFIRM_EMAIL_RESENT } },
+} = ROUTES;
+
+context('Insurance - Account - Create - Resend confirm email page - As an Exporter I want to request a new link to confirm my email address, So that I can readily use my email address to set up an account that I can use for UKEF digital service such as EXIP digital service', () => {
+  before(() => {
+    cy.navigateToUrl(START);
+
+    cy.submitEligibilityAndStartAccountCreation();
+    cy.completeAndSubmitCreateAccountForm();
+
+    confirmEmailPage.havingProblems.requestNew.link().click();
+  });
+
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('_csrf');
+    Cypress.Cookies.preserveOnce('connect.sid');
+  });
+
+  let expectedUrl;
+  let exporter;
+
+  describe('core page elements and content', () => {
+    before(() => {
+      /**
+       * Get the exporter ID directly from the API,
+       * so that we can assert that the URL and `request a new link` has the correct ID.
+       */
+      const exporterEmail = Cypress.env('GOV_NOTIFY_EMAIL_RECIPIENT');
+
+      api.getExporterByEmail(exporterEmail).then((response) => {
+        exporter = response;
+
+        expectedUrl = `${CONFIRM_EMAIL_RESENT}?id=${exporter.id}`;
```

Comment:
Can we please add void exporter check, in case api call fails.

Reply:
I don't think we need to, we handle error in `api.getExporterByEmail` - if there is API error/error calling the API, this function will throw an error and the E2E test will not continue.

As we progress the full application/accounts, want to avoid adding checks like this in actual tests as they'd be a lot of them